tests/utils.py

```diff
@@ -268,3 +299,25 @@ def all_pystack_combinations(corefile=False, native=False):
     [python[:2] for python in AVAILABLE_PYTHONS if python.version < (3, 10)],
     ids=[python[1].name for python in AVAILABLE_PYTHONS if python.version < (3, 10)],
 )
+
+
+def all_pythons_between(
+    *, from_version: Tuple[int, int], to_version: Optional[Tuple[int, int]] = None
+):
```

Comment:
This seems unnecessarily generic - you've defined a `all_pythons_between` that takes a mandatory start version and an optional end version, but you never provide that end version anywhere, so this feels like YAGNI. It seems like you could just do `all_pythons_since()` for now, and wait to define an `all_pythons_between` until we need it.